RecoParticleFlow/PFTracking/src/PFTrackAlgoTools.cc

```diff
@@ -0,0 +1,38 @@
+#include "RecoParticleFlow/PFTracking/interface/PFTrackAlgoTools.h"
+namespace PFTrackAlgoTools {
+unsigned int getAlgoCategory(const reco::TrackBase::TrackAlgorithm& algo,bool hltIterativeTracking) {
+  switch (algo) {
+  case reco::TrackBase::ctf:
+  case reco::TrackBase::duplicateMerge:
+  case reco::TrackBase::initialStep:
+  case reco::TrackBase::lowPtTripletStep:
+  case reco::TrackBase::pixelPairStep:
+  case reco::TrackBase::jetCoreRegionalStep:
+    return 0;
```

Comment:
Even just saying "highQuality", "mediumQuality, (since the Dpt/pt cut corresponds to the expected quality of the track), would be useful over cat0,cat1,cat2, or 0,1,2.
It doesn't have to be some deep philosophical concept.